cob_twist_controller/CMakeLists.txt

```diff
@@ -51,7 +53,7 @@ target_link_libraries(cob_twist_controller_node twist_controller ${catkin_LIBRAR
 
 ## Install
 # Mark executables and/or libraries for installation
-install(TARGETS cob_twist_controller_node twist_controller inverse_differential_kinematics_solver constraint_solvers damping_methods limiters inv_calculations
+install(TARGETS cob_twist_controller_node twist_controller inverse_differential_kinematics_solver constraint_solvers damping_methods limiters interface_types inv_calculations
```

Comment:
can we unify/harmonize the order of the libraries all during this document?
i.e. same order in catkin_package, add_library and install....